modules/upgrade-upgrading-cli.adoc

```diff
@@ -0,0 +1,147 @@
+you want to apply:
++
+----
+$ oc adm upgrade
+
+Cluster version is 4.0.0-0.nightly-2019-01-25-200832
+Updates:
+VERSION                       	IMAGE
+4.1.0-0.nightly-2019-01-25-201056 registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056
+4.1.0-0.nightly-2019-01-25-205123 registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-205123
+----
+
+. Apply an update:
+** To update to the latest version:
++
+----
+$ oc adm upgrade --to-latest=true <1>
+----
+
+** To update to a specific version:
++
+----
+$ oc adm upgrade --to=<version> <1>
+----
+<1> `<version>` is the update version that you obtained from the output of the
+previous command.
+
+. Review the status of the Cluster Version Operator:
++
+----
+$ oc get clusterversion -o json|jq ".items[0].spec"
+
+{
+  "channel": "stable-4.1",
+  "clusterID": "53f97452-9856-4a3c-8260-00c1de2668a1",
+  "desiredUpdate": {
+	"image": "registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056",
+	"version": "4.1.0-0.nightly-2019-05-25-201056" <1>
+  },
+  upstream": "https://api.openshift.com/api/upgrades_info/v1/graph"
+}
+----
+<1> If the `version` number in the `desiredUpdate` stanza matches the value that
+you specified, the update is in progress.
+
+. Review the cluster version status history to monitor the status of the update.
+It might take some time for all the objects to finish updating.
++
+----
+$ oc get clusterversion -o json|jq ".items[0].status.history"
+
+[
+  {
+	"completionTime": null,
+	"image": "registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056",
+	"startedTime": "2019-01-29T07:51:34Z",
+	"state": "Partial",
+	"version": "4.1.0-0.nightly-2019-05-25-201056"
+  },
+  {
+    "completionTime": "2019-05-20T15:47:21Z",
+    "image": "registry.svc.ci.openshift.org/ocp/ocp-release@sha256:6f4cf2db7e63c4dba54496a72b83fec22c49293b520ff0cdb78f1e38b23f1ccb",
+    "startedTime": "2019-05-20T15:23:29Z",
+    "state": "Completed",
+    "verified": false,
+    "version": "4.1.0-rc.4"
+  }
+]
+----
++
+The history contains a list of the most recent versions applied to the cluster.
+This value is updated when the CVO applies an update. The list is ordered by
+date, where the newest update is first in the list. Updates in the history have
+state `Completed` if the rollout completed and `Partial` if the update failed
+or did not complete.
++
+[IMPORTANT]
+====
+If an upgrade fails, the Operator stops and reports the status of the failing
+component. Rolling your cluster back to a previous version is not supported.
+If your upgrade fails, contact Red Hat support.
+====
+
+. After the update completes, you can confirm that the cluster version has
+updated to the new version:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.1.0       True        False         2m        Cluster version is 4.1.0
+----
```

Comment:
Refer to https://github.com/openshift/openshift-docs/pull/13993/files#r286777865